ReactiveObjC/RACSubject.h

```diff
@@ -16,11 +16,14 @@ NS_ASSUME_NONNULL_BEGIN
 ///
 /// They're most helpful in bridging the non-RAC world to RAC, since they let you
 /// manually control the sending of events.
-@interface RACSubject : RACSignal <RACSubscriber>
+@interface RACSubject<__covariant ValueType> : RACSignal<ValueType> <RACSubscriber>
```

Comment:
Isn't `RACSubject` invariant (neither covariant nor contravariant) as it is both a source and a sink?